Havtorn/Source/Engine/Graphics/RenderManager.h

```diff
@@ -91,6 +91,26 @@ namespace Havtorn
 		PointLightCube,
 		Icosphere,
 		LineShape,
+		FlatArrow,
+		DebugCube,
+		Camera,
+		CircleXY8Segments,
+		CircleXY16Segments,
+		CircleXY32Segments,
+		//TODO: add more debug shape primitives
```

Comment:
maybe nice to write TODO.AG if you know you're going to do it yourself, if you wanna leave it open, this is fine